recipes/pycrystal/bs4.patch

```diff
@@ -0,0 +1,23 @@
+From 18b1e75b7a3bb797a7d75d56fad1a8733c5cf989 Mon Sep 17 00:00:00 2001
+From: Jan Janssen <jan-janssen@users.noreply.github.com>
+Date: Mon, 15 May 2023 10:58:52 -0600
+Subject: [PATCH] Replace bs4 with beautifulsoup4
```

Comment:
I opened a pull request to include these changes upstream https://github.com/tilde-lab/pycrystal/pull/22